src/ComplexMethod.java

```diff
@@ -22,7 +22,10 @@ public String intToEnglishValue(int i) {
         if (i > 6) {
             throw new NotImplementedException();
         }
+        if (i > 7) {
+            throw new NotImplementedException();
+        }
         return null;
     }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 8 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS109)